src/components/ConnectButtons.tsx

```diff
@@ -32,4 +33,4 @@ const ConnectButtons = () => {
   );
 };
 
-export { ConnectButtons };
+export default ConnectButtons;
```

Comment:
We have mostly used named exports everywhere. Is this changed to default for a reason?

Reply:
In server side component, if you import a component with `dynamic`, you can't use `import(...).then()` to access the named export